web/src/compose_actions.ts

```diff
@@ -415,8 +415,11 @@ export let start = (raw_opts: ComposeActionsStartOpts): void => {
         drafts.set_compose_draft_id(opts.draft_id);
     }
 
-    // Show a warning if topic is resolved
-    compose_validate.warn_if_topic_resolved(true);
```

Comment:
We just want to avoid calling this in an `if` condition using arguments passed to `start`.  The condition should be something like `if (opts_trigger != "forward_message")`

Reply:
Yeah, this approach won't work in that case. 

Here's an alternative approach. We track if the `input` or `start` is part of forward message in `compose_validate.is_processing_message_forward` as a boolean. If it is `true`,  we can do an early return inside `warn_if_topic_resolved` .

Does that sound good?